Add link on website to new PyNEST templates and update citations (#1351)
nest/nest-simulator: doc/community.rst

NEST Community
===============

Mailing List
-------------

The NEST users mailing list is intended to be a forum for questions on the usage
of NEST, the exchange of code and general discussions about NEST.  The philosophy
is that all users profit by sharing their experience. All NEST core developers
are subscribed to this list and will participate in the discussions as far as
time allows.

By subscribing to the mailing list you will also get notified of all NEST related events!

For more information, subscription instructions and access to the archives,
please visit https://www.nest-initiative.org/mailinglist/.

Open video meeting for users and developers
--------------------------------------------

Every two weeks, we have an open video meeting to discuss current issues and developments in NEST.

> We welcome our users to join who may have questions regarding their implementations or issues they want help solving.

Reviewer: ```suggestion
We welcome users with questions regarding their implementations or issues they want help solving to join.
```

This is an opportunity to have discussions in real time with developers.

Information for dates and how to join can be found on `our GitHub wiki <https://github.com/nest/nest-simulator/wiki/Open-NEST-Developer-Video-Conference>`_

Publications using NEST
-------------------------

We have compiled  a `list of NEST-related peer-reviewed publications <https://www.nest-simulator.org/publications/>`_ that
we update regularly.

If you have used NEST in your research, let us know!

> Don't forget to :doc:`cite NEST <citing-nest>` in your work.

Reviewer: After the change of the Headline in the first line of this PR, should this not rather be the following?
```suggestion
Don't forget to :doc:`cite NEST <cite-nest>` in your work.
```

Author: citing-nest is the name of the file and not the header name
